prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -532,6 +534,14 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .toList()
     }
 
+    /**
+     * for the Message Tracker feature: Fetch ActionLogs by a report id and a filter type
```

Comment:
Instead of "for the Message Tracker feature" maybee "used by the Message Tracker feature"